### ISSUE TITLE
Training data for tokenizer

### ISSUE THREAD

**Issue by HalvesChen** [ByteFlow-AI/TokenFlow#19]
Hello, thank you for your excellent work. 
I would like to know the amount of training data you use when training the tokenizer? And, have you conducted ablation experiments on the data size.

**Comment by QuLiao1117** [comment]
Thanks for your attention to TokenFlow.

The tokenizers were trained on approximately 100M samples. We found that this volume of training data was sufficient for both semantic and reconstruction losses to converge. More training would not yield significant improvements.

**Comment by HalvesChen** [comment]
Thank you for your reply. 
Could you tell me the number of epochs you trained? It seems to be a little different from the paper and the code.

**Comment by QuLiao1117** [comment]
The current codebase contains the training code on ImageNet. You can train the model on ImageNet for **~100** epochs. The multimodal understanding capabilities are comparable to models trained on LAION and COYO datasets. For image reconstruction on the ImageNet validation set, models trained on ImageNet would perform better compared to those trained on other datasets. You can also opt to use LAION & COYO as your training dataset and adjust the training steps accordingly.

The main reason we chose to train on LAION and COYO datasets was to enhance the model's reconstruction capability on real-world images, which in turn improves the text-to-image generation performance.

**Comment by HalvesChen** [comment]
Thank you for your reply~
So what I understand about the training process is to train the semantic branch and pixel branch 100epoch on imagenet firstly, and then fine-tune on LAION and COYO100M data?

**Comment by QuLiao1117** [comment]
> Thank you for your reply~ So what I understand about the training process is to train the semantic branch and pixel branch 100epoch on imagenet firstly, and then fine-tune on LAION and COYO100M data?

You only need to train for ~100M images using either ImageNet data or LAION & COYO-700M datasets. The resulting performance will be approximately the same.

**Comment by HalvesChen** [comment]
Got it~
